### ISSUE TITLE
Traceback when starting a fresh install of JupyterLab (on old Node version)

### ISSUE THREAD

**Issue by sccolbert** [jupyterlab/jupyterlab#7353]
On a fresh `conda install jupyterlab` on Python 3.7, I get the following traceback when JLab starts:

```
(akadio) Ss-MacBook-Pro:firefly-extension Chris$ jupyter lab
[I 15:11:28.058 LabApp] JupyterLab extension loaded from /Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab
[I 15:11:28.058 LabApp] JupyterLab application directory is /Users/Chris/miniconda3/envs/akadio/share/jupyter/lab
[I 15:11:28.061 LabApp] Serving notebooks from local directory: /Users/Chris/Development/hdfgroup/hdflab/packages/firefly-extension
[I 15:11:28.061 LabApp] The Jupyter Notebook is running at:
[I 15:11:28.061 LabApp] http://localhost:8888/?token=33b1002f71da2df6f340634cc8132ff326d00192da785952
[I 15:11:28.061 LabApp]  or http://127.0.0.1:8888/?token=33b1002f71da2df6f340634cc8132ff326d00192da785952
[I 15:11:28.061 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 15:11:28.066 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///Users/Chris/Library/Jupyter/runtime/nbserver-3446-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/?token=33b1002f71da2df6f340634cc8132ff326d00192da785952
     or http://127.0.0.1:8888/?token=33b1002f71da2df6f340634cc8132ff326d00192da785952
[I 15:11:30.376 LabApp] 301 GET /lab/workspaces/auto-3/?clone (::1) 1.35ms
[E 15:11:31.759 LabApp] Uncaught exception GET /lab/api/build?1570997491526 (::1)
    HTTPServerRequest(protocol='http', host='localhost:8888', method='GET', uri='/lab/api/build?1570997491526', version='HTTP/1.1', remote_ip='::1')
    Traceback (most recent call last):
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab/commands.py", line 1657, in _node_check
        output = subprocess.check_output([node, 'node-version-check.js'], cwd=HERE)
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/subprocess.py", line 395, in check_output
        **kwargs).stdout
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/subprocess.py", line 487, in run
        output=stdout, stderr=stderr)
    subprocess.CalledProcessError: Command '['/usr/local/bin/node', 'node-version-check.js']' returned non-zero exit status 1.
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/tornado/web.py", line 1699, in _execute
        result = await result
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
        yielded = self.gen.throw(*exc_info)  # type: ignore
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab/handlers/build_handler.py", line 117, in get
        data = yield self.builder.get_status()
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
        value = future.result()
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/tornado/gen.py", line 742, in run
        yielded = self.gen.throw(*exc_info)  # type: ignore
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab/handlers/build_handler.py", line 39, in get_status
        self.app_dir, self.log, self.core_config)
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/tornado/gen.py", line 735, in run
        value = future.result()
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/concurrent/futures/thread.py", line 57, in run
        result = self.fn(*self.args, **self.kwargs)
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab/handlers/build_handler.py", line 92, in _run_build_check
        app_dir=app_dir, logger=logger, core_config=core_config)
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab/commands.py", line 419, in build_check
        _node_check(logger)
      File "/Users/Chris/miniconda3/envs/akadio/lib/python3.7/site-packages/jupyterlab/commands.py", line 1660, in _node_check
        data = CoreConfig()._data
    NameError: name 'CoreConfig' is not defined
```
CC @vidartf

**Comment by sccolbert** [comment]
This is `jupyterlab@1.1.4`

**Comment by sccolbert** [comment]
The issue also exists in `jupyterlab@1.1.3`

My system node version is `6.9.4`

**Comment by sccolbert** [comment]
Updating the system node to `10.16.3` removes the traceback.

Seems that something in the node version checker is failing, and that fail condition is not properly handled.

**Comment by vidartf** [comment]
This was fixed in https://github.com/jupyterlab/jupyterlab/pull/7079, but was not back-ported to 1.1.x. I'll leave it to the release managers to decide whether this should be backported to 1.1.x or not.

**Comment by vidartf** [comment]
I think maybe a partial fix for 1.1.x (adding the import) might be cleaner?

**Comment by vidartf** [comment]
Fix opened in https://github.com/jupyterlab/jupyterlab/pull/7374, in case we are still doing patches for 1.1.x.
